pkg/volume/emptydir/empty_dir_quota.go

```diff
@@ -12,8 +11,9 @@ var _ volume.Mounter = &emptyDirQuotaMounter{}
 
 // EmptyDirQuotaPlugin is a simple wrapper for the k8s empty dir plugin mounter.
 type EmptyDirQuotaPlugin struct {
-	// wrapped is the actual k8s emptyDir volume plugin we will pass method calls to.
-	Wrapped volume.VolumePlugin
+	// the actual k8s emptyDir volume plugin we will pass method calls to.
+	// TODO: do we need to implement unmount
```

Comment:
We should be ok without unmount, you can't actually remove an XFS quota, all we could do is zero it out but it's harmless to leave it, the plugin will just re-adjust it correctly if that GID were to ever land on this node again, otherwise it just sits there unused.